src/LineBreakTransformer.ts

```diff
@@ -1,14 +1,14 @@
-export class LineBreakTransformer {
-    #container = '';
+export class LineBreakTransformer implements Transformer<string, string> {
+    private container = '';
 
-    transform(chunk, controller) {
-        this.#container += chunk;
-        const lines = this.#container.split('\n');
-        this.#container = lines.pop();
+    transform(chunk: string, controller: TransformStreamDefaultController<string>) {
+        this.container += chunk;
+        const lines = this.container.split('\n');
+        this.container = lines.pop();
         lines.forEach((line) => controller.enqueue(line));
     }
 
-    flush(controller) {
-        controller.enqueue(this.#container);
+    flush(controller: TransformStreamDefaultController<string>) {
+        controller.enqueue(this.container);
```

Comment:
Add `this.container = ''` after flush?

Reply:
Generally the examples I've found [for transformers](https://developer.mozilla.org/en-US/docs/Web/API/TransformStream/TransformStream) don't do that. Ones I have seen tend to terminate the controller, that is, close the written-to stream, which this doesn't do. I haven't caught any bugs related to this particular bit of code so far so I'm tempted to leave it as-is and keep this in mind as I start looking at better unit test coverage.